drivers/vispdats/app/models/vispdats/synthetic/base.rb

```diff
@@ -0,0 +1,71 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Vispdats::Synthetic
+  class Base < ::GrdaWarehouse::Synthetic::Assessment
+    include ArelHelper
+
+    validates_presence_of :source
+
+    def assessment_date
+      source.submitted_at&.to_date
+    end
+
+    def assessment_location
+      source.user&.agency&.name
+    end
+
+    def assessment_type
+      # TODO add to the VI-SPDAT
+      3 # In Person
```

Comment:
Isn't this what you just added with contact_method?

Reply:
Good catch! Added.